mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -93,6 +110,10 @@ CLASS net/minecraft/unmapped/C_etbdktdp net/minecraft/entity/passive/VillagerEnt
 		ARG 0 villager
 		ARG 1 poiType
```

Comment:
POIType

Reply:
The reasonn POI should be capitalized is because it’s an acronym for Point of Interest therefore it’s best if everything is in caps and not just Poi otherwise it looks weird